### DISCUSSION TITLE
Need a feedback on my application how can i optimise my application made on mern stack technology

### DISCUSSION THREAD

**Discussion by coderAdityasen** [community/community#124626] [Programming Help]
### Select Topic Area

Product Feedback

### Body

hey there i need your valuable feedback on my application how can i optimize my ecommerce application have a look at my gitthub repo and give it a star if you like my work i am web dev enthusiast and build a project as a beginner this is my first complete full stack project on mern stack an ecommerce with admin dashboard and implement a ai chat bot by google i do not add any kind of designing i only focus on concept and functionality of the application 

https://github.com/coderAdityasen/full-stack-ecommerce

**Comment by RafaelJohn9** [comment]
hello, I have checked your site. It's a nice portfolio project. The documentation on the README  its also very nice and very well detailed, I like it
However
I noticed that it's not responsive in different devices like mobile devices
![image](https://github.com/community/community/assets/125447154/59d19594-5998-4806-a6f7-988d5aa3a8ca)
its an easy fix
You can check out videos on that its an easy concept considering you made a full-stack website. Congrats

**Reply by coderAdityasen** [reply]
yup its too easy to fix it : ) 
thanks for your feedback

**Comment by RafaelJohn9** [comment]
gave it a star

**Reply by coderAdityasen** [reply]
thankyou so much
